site/content@beta/backend-apis/apis-overview.md

```diff
@@ -0,0 +1,73 @@
+---
+weight: 30
+---
+
+# APIs Overview
+
+There are two primary Chat APIs:
+
+- [Agent Chat API](https://developers.livechatinc.com/beta-docs/agent-chat-api/), which serves to join a chat as an agent.
+- [Customer Chat API](https://developers.livechatinc.com/beta-docs/customer-chat-api/), which serves to join a chat as a customer.
+
+The **Agent Chat API** contains a different set of methods than the **Customer Chat API**. We separated both APIs as they cover different use cases. 
+
+To use those APIs, you need to be _authorized_. This topic is thoroughly explained in the [**Authorizing API calls**](../authorization) document.
+
+### RTM API vs. Web API
+
+Agent Chat API and Customer Chat API can be used either as **Real Time Messaging APIs (RTM APIs)** or **Web API**. 
+
+To learn about differences between these two API types, see the comparison below:
+
+
+|**Characteristics**|     **RTM API**      | **Web API**               |
+|:-----------------:|:--------------------:|:-------------------------:|
+|**connection type**|   stateful  e.g. websocket        |  stateless, via XHR requests     |
+| **finds out about state changes via**  |   [pushes](#pushes)      |    [webhooks](#webhooks)   |
+|**used by** |Agent App, Chat Widget|external apps, integrations|
+
+
+<!-- RTM API enables real-time communication. Websocket transport allows for **pushes**, which are server-client methods used to keep application state up-to-date. In case of the Agent App and the Chat Widget, the continuous connection is crucial. That's why they both implement RTM API. 
```

Comment:
these descriptions will be in dedicated documents: Agent/Customer Chat RTM API and Agent/Customer Chat Web API